src/ansys/mapdl/core/mapdl.py

```diff
@@ -3204,9 +3204,29 @@ def run(
         if "\n" in command or "\r" in command:
             raise ValueError("Use ``input_strings`` for multi-line commands")
 
+        # Checking kwargs
+        verbose = kwargs.pop("verbose", False)
+        save_fig = kwargs.pop("savefig", False)
+
         # check if we want to avoid the current non-interactive context.
```

Comment:
```suggestion
        # Check if you want to avoid the current non-interactive context.
```
Also, it says to use "you" rather than "we".